services/workflows-service/src/case-management/types/transition-schema.ts

```diff
@@ -0,0 +1,5 @@
+export interface TransitionSchema {
```

Comment:
Is there a schema we can infer the type from? Having an interface named like a schema is a bit confusing.
```suggestion
export interface ITransitionSchema {
```